vector/src/main/java/im/vector/app/features/roomprofile/settings/RoomSettingsController.kt

```diff
@@ -145,5 +150,54 @@ class RoomSettingsController @Inject constructor(
                 id("guestAccessDivider")
             }
         }
+
+        // Security
+        buildProfileSection(stringProvider.getString(R.string.room_profile_section_security))
```

Comment:
I do not think this is the correct place to add this new security settings.
I the up screen there is already a "Security" section ([there](https://github.com/vector-im/element-android/blob/main/vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileController.kt#L99)). Maybe it will be better to put this over there.